python/gdal_select/Portfile

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           stub 1.0
+PortGroup           select 1.0
+
+name                gdal_select
+version             0.1
+revision            1
```

Comment:
revision field starts at zero but the `py-gdal` port needs a `revision` increase as it now installs "select" files.